<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debugger</class>
 <widget class="QWidget" name="Debugger">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>629</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
  width: 175px;
  height: 30px;
  background-color: #4a64ff;
  border-radius: 4px;

  font-family: ArialMT;
  font-size: 14px;
  font-weight: normal;
  font-style: normal;
  text-align: center;
  color: #ffffff;

}


QPushButton:hover
{
	 background-color: #4a34ff;
}



QCheckBox::indicator {
  border: 2px solid rgb(74, 100, 255);
  border-radius: 4px;
  width: 16px;
  height: 16px;
  subcontrol-position: left;
}

QCheckBox::indicator:checked {
  background-color: rgb(74, 100, 255);
}

QLabel {
margin-top: 6px;
background-color: transparent;
font: bold 16px &quot;monospace&quot;;
}

QScrollBar:horizontal {
  background: #262628;
}

QScrollBar::handle:horizontal {
  background: #4a4a4b;
  border: 0;
  border-radius: 3px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}

QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
  border: none;
  background: none;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="DebugWidget_Top" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <property name="horizontalSpacing">
             <number>15</number>
            </property>
            <property name="verticalSpacing">
             <number>10</number>
            </property>
            <item row="3" column="0">
             <widget class="QLabel" name="IIODeviceAttributesLabel">
              <property name="text">
               <string>IIO Device Attributes</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QLabel" name="FilenameLabel">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Filename</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="4">
             <widget class="QLineEdit" name="filenameLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="ValueLabel">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Value:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QComboBox" name="AttributeComboBox">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="ChannelComboBox"/>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="WriteButton">
              <property name="text">
               <string>Write</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="DeviceSelectionLabel">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Device Selection</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="DeviceLabel">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Device</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="DevicecomboBox"/>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="ReadButton">
              <property name="text">
               <string>Read</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QStackedWidget" name="valueStackedWidget">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <widget class="QLineEdit" name="valueLineEdit">
                <property name="geometry">
                 <rect>
                  <x>4</x>
                  <y>-6</y>
                  <width>191</width>
                  <height>31</height>
                 </rect>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="page_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <widget class="QComboBox" name="valueComboBox">
                <property name="geometry">
                 <rect>
                  <x>-2</x>
                  <y>-6</y>
                  <width>201</width>
                  <height>30</height>
                 </rect>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 87, 83);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="DebugWidget_Bottom" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>10</number>
            </property>
            <item row="3" column="1">
             <widget class="QCheckBox" name="detailedRegMapCheckBox">
              <property name="text">
               <string>Detailed Register Map</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="addressLabel">
              <property name="styleSheet">
               <string notr="true">color: rgb(238, 238, 236);</string>
              </property>
              <property name="text">
               <string>Address:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="3">
             <widget class="QLabel" name="addressHexLabel">
              <property name="text">
               <string>0x0000</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="sourceComboBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="regMapSettingsLabel">
              <property name="text">
               <string>Register Map Settings</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="displayModeLabel">
              <property name="text">
               <string>Display mode:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="9" column="3">
             <widget class="QLabel" name="valueHexLabel">
              <property name="text">
               <string>&lt;value&gt;</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QSpinBox" name="addressSpinBox">
              <property name="prefix">
               <string>0x</string>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="displayIntegerBase">
               <number>16</number>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="readRegPushButton">
              <property name="text">
               <string>Read</string>
              </property>
             </widget>
            </item>
            <item row="6" column="4">
             <widget class="QLabel" name="descriptionLabel">
              <property name="text">
               <string>Description:</string>
              </property>
             </widget>
            </item>
            <item row="9" column="4">
             <widget class="QLabel" name="defaultValueLabel">
              <property name="text">
               <string>Default Value:</string>
              </property>
             </widget>
            </item>
            <item row="9" column="2">
             <widget class="QSpinBox" name="valueSpinBox">
              <property name="prefix">
               <string>0x</string>
              </property>
              <property name="maximum">
               <number>429496729</number>
              </property>
              <property name="displayIntegerBase">
               <number>16</number>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QCheckBox" name="enableAutoReadCheckBox">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Enable AutoRead</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QPushButton" name="writeRegPushButton">
              <property name="text">
               <string>Write</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">color: rgb(238, 238, 236);</string>
              </property>
              <property name="text">
               <string>Value:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="sourceLabel">
              <property name="text">
               <string>Source:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="5">
             <widget class="QLineEdit" name="descriptionLineEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>170</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
